src/modules/collections/services/collections.service.ts

```diff
@@ -86,6 +90,18 @@ export class CollectionsService {
 		};
 	}
 
+	public adaptByVisitEndAt(visits: Pick<Visit, 'endAt'>[]): string | null {
```

Comment:
the "adapt" is usually used to give it an object and get a similar but modified object back.
I would rename this to
```suggestion
	public getLastEndAtDate(visits: Pick<Visit, 'endAt'>[]): string | null {
```

Reply:
Adjusted